mafia_bot.py

```diff
@@ -34,16 +34,9 @@ def start_up(path,url):
     return(driver)
 
 def log_in(driver,username,password):
-    account_box = driver.find_element_by_class_name('account-box')
-    user_entry = account_box.find_elements_by_class_name('account-input')[0]
-    outerHTML = user_entry.get_attribute('outerHTML')
-    id = outerHTML[outerHTML.index('label for=')+11:][:18]
-    login = driver.find_element_by_id(id)
-    login.send_keys(username)
-    time.sleep(0.5)
-    login = driver.find_element_by_id(id[:-1]+'2')
-    login.send_keys(password)
-    login.send_keys('\n')    
+    driver.find_element_by_xpath("//input[@type='text']").send_keys(username);
+    driver.find_element_by_xpath("//input[@type='password']").send_keys(password);
+    driver.find_element_by_xpath("//button[@type='submit']").click(); 

```

Comment:
can we add a check here to figure out whether the login was successful?
and raise an error if it failed

Reply:
and then when we move this to raw api it would be easier to write unit tests for this

Reply:
uh i think i updated my branch to have logging now but can't tell since it doesn't update the pull request...?